src/Integration.Vsix.UnitTests/SonarAnalyzerCollisionManagerTests.cs

```diff
@@ -0,0 +1,163 @@
+//-----------------------------------------------------------------------
+// <copyright file="BoundSolutionAnalyzerTests.cs" company="SonarSource SA and Microsoft Corporation">
```

Comment:
Note to self: check all copyright headers that were added in this PR, because all of them has been blindly copied.